.changeset/young-coats-tie.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-admin": patch
+---
+
+Adapt styling of the search input icons in `PageSearch` to match the Comet DXP design
```

Comment:
`PageSearch` is an internal component. I'd change the changeset to name the feature instead:

```suggestion
Adapt styling of the page tree search input to match the Comet DXP design
```